tests/test_vpc.py

```diff
@@ -8,6 +8,18 @@
 from botocore.exceptions import ClientError as BotoClientError
 from c7n.exceptions import PolicyValidationError
 from c7n.resources.aws import shape_validate
+from pytest_terraform import terraform
+
+
+@terraform('sg')
```

Comment:
please use a descriptive name for the fixture, `sg` is not descriptive, and additionally there's too many components bundled into it for it to be reusable. I'd suggest separating it out it to aws_network as a session fixture, and a separate aws_code_build_vpc terraform fixture.

Reply:
Ah! Thanks for the pointers.

Reply:
I renamed the existing fixture to `aws_code_build_vpc`, without splitting up into two.

Why? My attempt to add two separate fixtures resulted in an error whereas the resourced defined in the first fixture could not be looked up by the second fixutre. This was when I attempted to simply stack the two fixtures, maybe like so:

```
@terraform('aws_network', scope='session')
@terraform('aws_code_build_vpc')
```

Reply:
please file an issue on the pytest-terraform repo for this. in the interim, the fixture will need to be declared in a non decorator fashion I suspect.